MyCryptoHQ/MyCrypto
# Copy to clipboard

Closes #1000 

### Description

Allows users to copy their address, by clicking the copy icon. 

### Changes

Document/copy icon next to account address that copies it to the clipboard

### Steps to Test

1. Unlock wallet
2. Under account, address click copy icon
3. Paste in any text field to see if address was copied
